src/recipes/SupportedBrowser.js

```diff
@@ -0,0 +1,33 @@
+/* 
+    RECIPE: SupportedBrowserPage
+    -------------------------------------------------------------
+    Author: Malick Sere, Lia Hiscock, Joel Ramos, Morgan Graham
+    Description: This recipe looks for strings that would indicate that a page is a "supported browser" page.
+*/    
+
+
+
+void function() {
+    window.CSSUsage.StyleWalker.recipesToRun.push( function SupportedBrowserPage( element, results) {
+        
+        if(element.nodeName == "H1" || element.nodeName == "TITLE")
```

Comment:
Wait, you only want to look at `<h1>` or `<title>` elements? This is making a very large assumption on markup that you shouldn't. If you're trying to avoid looking into certain tags to lower false positives I'd do the inverse and check that they aren't `<script>`, `<html>`, `<body>` tags.